code/game/machinery/vitals_monitor.dm

```diff
@@ -50,76 +72,157 @@
 				breathing = "normal"
 			else if(lungs.breath_fail_ratio < 1)
 				breathing = "shallow"
-		
+
 		to_chat(user, SPAN_NOTICE("Breathing: [breathing]"))
+	if(connected_optable)
+		to_chat(user, SPAN_NOTICE("Connected to adjacent [connected_optable]."))
 
 /obj/machinery/vitals_monitor/Process()
 	if(QDELETED(victim))
-		victim = null
+		update_victim()
 	if(victim && !Adjacent(victim))
-		victim = null
-		update_use_power(POWER_USE_IDLE)
+		update_victim()
```

Comment:
This whole proc is calling `update_victim()` repeatedly.

Reply:
It should be noted the only time this calls update_victim() repeatedly is on QDELETED(victim), which is triggered when either victim is marked for qdel, or victim is null.